apps/ewallet/lib/ewallet/web/v1/error_handler.ex

```diff
@@ -315,9 +315,9 @@ defmodule EWallet.Web.V1.ErrorHandler do
   end
 
   defp stringify_errors(changeset, description) do
-    Enum.reduce(changeset.errors, description, fn {field, {description, _values}}, acc ->
+    Enum.reduce(changeset.errors, description, fn {field, {description, values}}, acc ->
       field = field |> stringify_field() |> replace_uuids()
-      acc <> " " <> field <> " " <> description <> "."
+      acc <> " " <> field <> " " <> replace_placeholders(description, values) <> "."
```

Comment:
We already have `build_template()` above that does something similar. Could you maybe use it and change it to `%{...}` format instead so we align with `Ecto.Changeset`'s errors?